api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -2202,4 +2204,37 @@ public void deleteChildAccount(final String id) throws UnauthorizedException, Ge
         System.out.println("url: " + url);
         messageBirdService.deleteByID(url, id);
     }
+
+    /**
+     * Returns outbound pricing for the default SMS configuration for the authenticated account.
+     *
+     * @return outbound pricing for the default SMS configuration for the authenticated account
+     *
+     * @throws UnauthorizedException if client is unauthorized
+     * @throws GeneralException      general exception
+     * @throws NotFoundException     if pricing information could not be found
+     *
+     * @see <a href="https://developers.messagebird.com/quickstarts/pricingapi/list-outbound-sms-prices/">Pricing API</a>
+     */
+    public OutboundSmsPriceResponse getOutboundSmsPrices() throws GeneralException, UnauthorizedException, NotFoundException {
+        return messageBirdService.request(OUTBOUND_SMS_PRICING_PATH, OutboundSmsPriceResponse.class);
+    }
+
+    /**
+     * Returns outbound SMS pricing for a specific SMPP username.
+     *
+     * @param smppUsername the SMPP SystemID provided by MessageBird
+     *
+     * @return outbound SMS pricing for the given SMPP username
+     *
+     * @throws UnauthorizedException if client is unauthorized
+     * @throws GeneralException      general exception
+     * @throws NotFoundException     if pricing information could not be found for the given SMPP username
+     *
+     * @see <a href="https://developers.messagebird.com/quickstarts/pricingapi/list-outbound-sms-prices/">Pricing API</a>
+     */
+    public OutboundSmsPriceResponse getOutboundSmsPrices(final String smppUsername) throws GeneralException, UnauthorizedException, NotFoundException {
+        final String url = String.format(OUTBOUND_SMS_PRICING_SMPP_PATH, smppUsername);
```

Comment:
can you add null check here for smppUserName? ```if (smppUsername == null) {
            throw new IllegalArgumentException("SMPP username must be specified.");
        }```

Reply:
Done in b6a4f90.